.github/workflows/_link_check.yml

```diff
@@ -7,14 +7,10 @@ on:
       ref:
         type: string
         required: true
-      run-url-lint:
-        type: boolean
-        required: false
```

Comment:
Should we keep it non-requires so that even if it crashes it won't block things?

Reply:
With the current change it'll always succeed until we make sure it's stable enough and let it fail again. And now having it run for each PR helps with experiments to collect logs.